lib/node.js

```diff
@@ -353,9 +353,11 @@ declare module "crypto" {
   ): void;
 }
 
+type address = {address: string; family: string; port: number};
```

Comment:
everything in libs is global, so we need to give this a more specific name. `dgram$Socket$address`, perhaps?